mmdet/models/backbones/resnet.py

```diff
@@ -302,7 +345,7 @@ def forward(self, x):
 
     def train(self, mode=True):
```

Comment:
If we freeze BN parameters and set eval mode in `build_norm_layer`, the `train()` method can be significantly simplified.